tests/storage-volumes-vm

```diff
@@ -254,6 +254,10 @@ do
 		lxc storage volume attach "${poolName}" virtual-machine/v2 v1 v2-root
 		sleep 3
 		lxc exec v1 -- test -L /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_lxd_v2--root
+
+		# Test that unmount succeeds; failed unmount will not cause VM stop to fail.
+		lxc storage volume detach "${poolName}" virtual-machine/v2 v1
```

Comment:
CI is unhappy about this.

Reply:
Yep, depends on https://github.com/canonical/lxd/pull/14929. Since that's merged we can rerun those for latest/edge once the build finishes.